tests/release/release-failures.go

```diff
@@ -0,0 +1 @@
+package release
```

Comment:
I don't get how this file relates to the rest of the PR

Reply:
Just wanted to preserve this go file as to keep it as a placeholder for the actual release-failures.go file to be added in full tests

Reply:
Shouldn't the PR adding the actual release-failures.go file in full tests add this file instead, then? Same comment for the next file. I think it makes the commit history much cleaner